app/.eslintignore

```diff
@@ -5,3 +5,4 @@ localization/messages
 localization/translated/en.json
 package.json
 package-lock.json
+src/pages/inside/logsPage/modals/attachmentModal/har/WithZipJs.jsx
```

Comment:
I think it's better to add eslint directive to the file itself instead of this line